docs/docs/cloud/cloud-cli.mdx

```diff
@@ -48,6 +48,90 @@ meltano cloud config env delete TAP_GITHUB_AUTH_TOKEN
 
 See the [reserved variables](/cloud/platform#reserved-variables) docs for more details on variables that are reserved for use by Meltano Cloud.
 
+### `notification`
+
+The `notification` command provides a way for managing your [Meltano Cloud notifications](concepts#meltano-cloud-notifications).
+
+#### `list`
+
+List all notifications configured for your current default project or organization.
+
+```sh
+$ meltano cloud config notification list
```

Comment:
@leoroese I know we're deep into this PR but how hard is it to show the filter on the notification list? 

My process was I did a new interactive notification but all it asked for was my email. Then when I do `list` I just see the active notification but with no idea what notifications I'm going to get.

Reply:
It's currently being returned, it would be super easy to display

Reply:
Looks good